app/search/flows/next-page-viewer-search.ts

```diff
@@ -31,16 +31,16 @@ export const nextPageViewerSearch = (): Thunk => (dispatch, getState) => {
 }
 
 function nextPageArgs(
-  logs: zng.Record[],
+  logs: zed.Record[],
   span: DateTuple
 ): [number, DateTuple] {
   let spliceIndex = 0
   const nextSpan: DateTuple = [...span]
   if (!isEmpty(logs)) {
     const index = indexOfLastChange(logs, (log) => log.try("ts")?.toString())
     if (index >= 0) {
-      const ts = logs[index].get("ts") as zng.Primitive
-      if (ts.getType() == "time") {
+      const ts = logs[index].try<zed.Time>("ts")
+      if (ts && ts instanceof zed.Time) {
```

Comment:
Would
```js
       if (ts instanceof zed.Time) {`
```
suffice here or is the `ts &&` necessary?

Reply:
You are right, it is not needed. Sweet.